test/unit/block_test.rb

```diff
@@ -9,29 +9,18 @@ def test_no_allocation_of_trimmed_strings
     assert_equal 3, template.root.nodelist.size
   end
 
-  def test_pre_trim
```

Comment:
This test and test_bug_compatible_pre_trim were moved to the shared liquid integration tests (https://github.com/Shopify/liquid/pull/1291/files#diff-abd62f5ce9d5df8c4bce6644f57a9b7a) so I removed them here.